pkg/syncer/util_test.go

```diff
@@ -7,105 +7,119 @@ import (
 	"testing"
 
 	"github.com/google/uuid"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Rename v1 to something else

Reply:
changed to `corev1 "k8s.io/api/core/v1"`